infrastructure/staging/network.tf

```diff
@@ -0,0 +1,7 @@
+module "network" {
+  source     = "../modules/network"
+  name       = "workflow"
```

Comment:
rename it to workflow-stage oder -staging maybe?

Reply:
renamed